inc/environment_indicator/namespace.php

```diff
@@ -88,6 +89,32 @@ function add_admin_bar_dashboard_link( WP_Admin_Bar $wp_admin_bar ) {
 	] );
 }
 
+/**
+ * Add menu item to the Altis logo in the admin bar for creating a support ticket.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar WP_Admin_Bar instance, passed by reference.
+ */
+function add_admin_bar_support_ticket_link( WP_Admin_Bar $wp_admin_bar ) {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	$env_name    = get_environment_name();
+	$support_url = 'https://dashboard.altis-dxp.com/#/support/new';
+	$support_url = 'unknown' === $env_name ? $support_url : "{$support_url}?applications[]={$env_name}";
```

Comment:
Ternary operators are a bit harder to read in general. In this instance you're not actually modifying the URL when the condition is true so a basic `if` statement would be a better fit and easier to understand:
```suggestion
	if ( $env_name !== 'unknown' ) {
		$support_url .= sprintf( '?applications[]=%s', urlencode( $env_name ) );
	}
```